packages-next/admin-ui/src/components/Navigation.tsx

```diff
@@ -62,21 +62,6 @@ const NavItem: FunctionComponent<NavItemProps> = ({ href, children }) => {
   );
 };
 
-const SignoutButton: FunctionComponent = () => {
```

Comment:
Haha I just pulled this down to fix it since that was my bad, you beat me to it 😁